scripts/install_script_ubuntu.sh

```diff
@@ -211,17 +211,22 @@ fi
 header "Initialize the configuration, building clients and setting options."
 run_cmd_confirm grr_config_updater initialize
 
-header "Enable grr-single-server to start automatically on boot"
-SERVER_DEFAULT=/etc/default/grr-single-server
-run_cmd_confirm sed -i 's/START=\"no\"/START=\"yes\"/' ${SERVER_DEFAULT};
-
-header "Starting up the service"
-initctl status grr-single-server | grep "running"
-IS_RUNNING=$?
-if [ $IS_RUNNING = 0 ]; then
-  run_cmd_confirm service grr-single-server stop
-fi
-run_cmd_confirm service grr-single-server start
+header "Enable grr services to start automatically on boot"
+
+for SERVER in grr-http-server grr-worker grr-ui
+do
+  SERVER_DEFAULT=/etc/default/${SERVER}
+  run_cmd_confirm sed -i 's/START=\"no\"/START=\"yes\"/' ${SERVER_DEFAULT};
```

Comment:
It may be better to leave them as No just for a sane default when installing additional GRR servers.  We install them a bit differently so this wouldn't directly impact how we do it, but just a thought since this script is for installing everything on a single host where GRR at scale is distributed.